<?xml version="1.0"?>
<launch>

  <node pkg="image_publisher" type="image_publisher"
    name="image_pub" output="screen"
    args="$(find image_manip)/data/lava.jpg" >
    <param name="publish_rate" value="10.0"/>
  </node>

  <node pkg="rsc_test" type="image_sub_pub"
    name="image_sub_pub" output="screen" >
    <remap from="image_in" to="image_pub/image_raw" />
    <remap from="image_out" to="image_out_rs" />
  </node>

  <node pkg="rsc_test" type="image_sub_pub_cpp"
    name="image_sub_pub_cpp" output="screen" >
    <remap from="image_in" to="image_pub/image_raw" />
    <remap from="image_out" to="image_out_cpp" />
  </node>

</launch>
